src/otx/core/model/segmentation.py

```diff
@@ -148,7 +167,7 @@ def __init__(
         self.decode_head_configuration = decode_head_configuration if decode_head_configuration is not None else {}
         export_image_configuration = export_image_configuration if export_image_configuration is not None else {}
         self.criterion_configuration = criterion_configuration
-        self.image_size = tuple(export_image_configuration.get("image_size", (1, 3, 512, 512)))
+        self.image_size = tuple([int(i) for i in export_image_configuration.get("image_size", (1, 3, 512, 512))])
```

Comment:
Does `image_size` in `export_image_configuration` have non-integer values?

Reply:
I tried to introduce a more strict type control here, but explicit conversion doesn't make mypy happy, so I just omitted type annotation in `OTXSegmentationModel`. Can be reverted here

Reply:
For safety, forcing type conversion seems fine. Could you update type annotation for `self.image_size` to `tuple[int, ...] | None` at L40?
https://github.com/openvinotoolkit/training_extensions/blob/9a1e80092d26162326dfeb15a051d451da956f97/src/otx/core/model/segmentation.py#L40

Reply:
Int tuple type was restored